account_financial_risk/models/account_invoice.py

```diff
@@ -71,8 +71,8 @@ def post(self):
             if exception_msg:
                 # When we post from validate.account.move wizard, validate_move returns
                 # 'ir.actions.act_window_close'
-                # Check active model to raise exception instead of returns wizard
-                if self.env.context.get("active_model") == "account.move":
+                # Active active_model is false in header buttons as Post in invoice form view
+                if not self.env.context.get("active_model", False):
```

Comment:
I think it should be this for covering the previous case:

```suggestion
                if self.env.context.get("active_model", "account.move") == "account.move":
```

Reply:
When `post` method is called from validate wizard active_model is "account.move", because wizard is launched from action.

https://github.com/odoo/odoo/blob/26217a1863d08e5aae967246da558aee8b0a32f3/addons/account/wizard/account_validate_account_move.py#L20
https://github.com/odoo/odoo/blob/26217a1863d08e5aae967246da558aee8b0a32f3/addons/account/wizard/account_validate_account_move.py#L10

Reply:
Then I don't get which cases should go by one path or another. I thought you wanted the "interactive" popup for that case and the manual button.

Reply:
Any answer to this?